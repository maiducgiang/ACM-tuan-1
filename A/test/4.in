4
H H
F Y
E G
B G