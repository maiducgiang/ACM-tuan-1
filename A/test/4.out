H => H
F => G => H => Y
E => D => H => G
B => F => G
